src/ansys/dpf/core/operator_specification.py

```diff
@@ -429,7 +433,16 @@ def _fill_pins(self, binput, to_fill):
                     self._api.operator_specification_get_pin_type_name(self, binput, i_pin, i_type)
                     for i_type in range(n_types)
                 ]
-
+                pin_aliases = []
+                if hasattr(self._api, "operator_specification_get_pin_num_aliases"):
+                    for i_alias in range(
+                        self._api.operator_specification_get_pin_num_aliases(self, binput, i_pin)
+                    ):
+                        pin_aliases.append(
+                            self._api.operator_specification_get_pin_alias(
+                                self, binput, i_pin, i_alias
+                            )
+                        )
                 pin_derived_class_type_name = ""
                 if server_meet_version("7.0", self._server) and hasattr(
```

Comment:
When @MichaelNale added this we were checking that the method is in the api and that the server version was met. Now that I see your solution, the name check seems enough. Can you remove the server version check here? Thanks!

Reply:
It turns out this is necessary

Reply:
And why is it not needed for the alisases?